### PR TITLE
[Yosegi-5] Write Hadoop's quick start document.

### PR DESCRIPTION
**Pull request by koijima**
### What is the necessity of this update? What is updated?
It is an easy-to-use document on MapReduce.

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+Please create Hadoop installation and execution environment of MapReduce.
+
+[Apache Hadoop](https://hadoop.apache.org/)
+
+
+## Preparation of input data.
+In this example, JSON data is assumed to be input data.
+With this data as input, we create a table in Yosegi format and read data from the table.
+
+Create the following csv file.In this example, to illustrate writing to and reading from the table, the data is a simple example.
```
Comment (yohto) [inline docs/quickstart.md]: Create the following CSV file. Following example is for illustrating to write and read simple data from the table.

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+{"id":"X_0004","name":"DDD","age":21}
+{"id":"X_0005","name":"EEE","age":28}
+{"id":"X_0006","name":"FFF","age":21}
+__JSON__
+```
+
+## Create yosegi file
+Load json file and create yosegi file.
+
+The sample code for MapReduce is this.
```
Comment (yohto) [inline docs/quickstart.md]: Following code is a sample for MapReduce.

File: docs/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+This class is included in the jar.
```
Comment (yohto) [inline docs/quickstart.md]: This class should be included in the jar file.

File: docs/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+Run MapReduce.
+
+```
+CLASSPATH=./yosegi.jar yarn jar yosegi-hadoop-0.9.0.jar jp.co.yahoo.yosegi.hadoop.mapreduce.YosegiWriteMapReduceExample -libjars yosegi.jar /tmp/example.json  /tmp/example_yosegi
+```
+
+## Read yosegi file
+Read as a data frame from yosegi file.
+
+The sample code for MapReduce is this.
```
Comment (yohto) [inline docs/quickstart.md]: Following code is a sample for MapReduce.

File: docs/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+This class is included in the jar.
```
Comment (yohto) [inline docs/quickstart.md]: This class should be included in the jar file.

**Review by yohto — COMMENTED** [review]
Please check some English sentences i commented.

**Comment by koijima** [comment]
Thank you for your review.
Fixed English sentences.

**Comment by yohto** [comment]
LGTM

**Pull request merged**